index.html

```diff
@@ -144,10 +144,29 @@ <h2>Terminology</h2>
 <section id="object-spec">
     <h2>Object Specification</h2>
     <p>
-        An OCFL Object is a group of one or more content bitstreams (data and metadata), and their administrative
-        information that are together identified by a URI. The object may contain a sequence of versions of the
-        bitstreams that represent the evolution of the object's contents.
+        An OCFL Object is a group of one or more files, directories, and administrative
+        information, that are together identified by a URI. The object may contain a sequence of versions of the
+        files that represent the evolution of the object's contents.
     </p>
+    <p>
+        A file is defined as a content bitstream that can be stored and transmitted. Directories (also called "folders")
+        allow for the organization of files into tree-like hierarchies. The content of an OCFL Object is the files and
+        directories that are stored <i>within</i> the hierarchy layout provided as part of the spec.
+    </p>
+    <p>
+        The administrative information is specific to OCFL Objects. Implementers of OCFL are strongly encouraged to
+        store their own metadata serializations as files in an OCFL Object, but, in the context of the OCFL Specification
+        these are considered to be part of the contents of an OCFL Object.
+    </p>
+    <p>
+        An OCFL Object is therefore:
+    </p>
+    <ol>
+        <li>A conceptual gathering of all files (data and metadata), folders, and their changes over time which together
+            form the digital representation of a complete intellectual endeavour (i.e., content); and</li>
```

Comment:
I still feel that introducing the notion of a "complete intellectual endeavour", whatever that might mean, is spurious. We also need to decide on US or UK spelling and use that consistently: endeavor or endeavour. I think I would scratch everything from "An OCFL object is therefore:" onward -- if it isn't clear from the first portion then we should fix that instead of offering a second try

Reply:
My concern with not including something like a "intellectual endeavor" (US spelling or bust) is that folks aren't going to understand what to put in an OCFL object.  

We often had this argument in the early days of DPN.  Some folks felt that an object was an entire collection, others felt that it was a single book, while still others package up their entire repository.  

Contrast that with some of the staff here at my library who have argued that an object is a single file i.e. the PDF is an object and each one of the TIFs of the pages are also single objects.  The goal of OCFL -- in my mind at least -- is to say that an OCFL object is the book, not a collection of books nor the individual files that make up the book.

I realize this is flame war worthy, but like using US spelling, I'm willing to duke it out because I think this is an important conversation to be had.

Reply:
I don't think we should be too proscriptive about what is in an OCFL object - as long as it has enough metadata to describe itself then I don't see why there is a problem. In particular, I want to store people, places and collections as distinct objects. What about WARC files or disk images?

Reply:
Not to get us off into the weeds further, if we were to draw a parallel with PREMIS (http://www.loc.gov/standards/premis/v3/premis-3-0-final.pdf - just because I happen to look at that tody) then I think an OCFL object would map to a PREMIS digital Representation (minus the recursion)

The point about whether you have an OCFL object for a page, a book, or a set of books is entirely up to the repository and data managers in my opinion. OCFL is completely agnostic on the issue, the object is merely a set of files organized with a directory hierarchy (or not) that are managed as one lump, with versioning